big-practice/src/js/dashboard.js

```diff
@@ -7,13 +7,15 @@ const tableBody = document.getElementById('tableBody');
 const nameMovieInput = document.getElementById('nameMovie');
 const directorInput = document.getElementById('director');
 const nationInput = document.getElementById('nation');
-const createBtn = document.getElementById('createBtn');
-const cancelBtn = document.getElementById('cancelBtn');
+const formCreateBtn = document.getElementById('createBtn');
+const cancelBtn = document.querySelector('.cancel-btn');
 const accountName = document.querySelector('.account-name');
 const addBtn = document.querySelector('.add-btn');
-const modal = document.querySelector('.modal');
-const updateBtn = document.getElementById('updateBtn');
+const formUpdateBtn = document.getElementById('updateBtn');
 const form = document.querySelector('.form');
+const modalForm = document.querySelector('.modal-form');
+const modalWarning = document.querySelector('.modal-warning');
+const warningCancelBtn = document.querySelector('.warning-cancel-btn');
```

Comment:
Instead of using .warning-cancel-btn, is there any way to query this element without adding .warning-cancel-btn into button?